addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -700,7 +700,7 @@ options.registry.WebsiteSaleProductPage = options.Class.extend({
     },
 
     _updateWebsiteConfig(params) {
-        this._rpc({
+        return this._rpc({
             route: '/shop/config/website',
             params,
         }).then(() => this.trigger_up('request_save', {reload: true, optionSelector: this.data.selector}));
```

Comment:
Add a TODO next to those detected double reload:
- Reminder to remove them in master
- If we/a community dev stumble onto this code, we/he'll know it's not something to copy/paste.